egs/wsj_mix/asr1/RESULTS.md

```diff
@@ -3,21 +3,38 @@
 ## CER
```

Comment:
Maybe we can just provide the averaged results?
Also, is it possible to follow the result form used in the other recipes?
Please check https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#how-to-contribute-to-espnet

Reply:
The new version is updated.